docker/Dockerfile.nvidia

```diff
@@ -125,16 +116,14 @@ RUN if [ "x$MPI_VER" = "x4" ]; then \
         ln -sf $HPCSDK_HOME/comm_libs/hpcx/latest/ompi \
                $HPCSDK_HOME/comm_libs/mpi ; \
     fi;  \
-    /venv/bin/pip install --no-cache-dir -r /app/requirements-mpi.txt && \
-    rm -rf ~/.cache/pip
-ENV CFLAGS=
 
-ADD ./devito /app/devito
-ADD ./tests /app/tests
-ADD ./scripts /app/scripts
-ADD ./examples /app/examples
-ADD ./benchmarks /app/benchmarks
-COPY setup.cfg /app/
+# install
```

Comment:
Maybe enrich...
# Upgrade pip, install wheel, devito and rest....

Reply:
Fixed